src/himali/Assignment24/Program2.java

```diff
@@ -0,0 +1,35 @@
+package himali.Assignment24;
+
+public class Program2 {
+	void getNumberSum(String a) {
+		String s = "";
+		int sum = 0;
+		for (int i = 0; i < a.length(); i++) {
+			char ch = a.charAt(i);
+
+			if (Character.isDigit(ch)) {
+				s = s + ch;
+
+			} else if (s.isEmpty()) {
+				continue;
```

Comment:
This is good,

I feel you also check this code as well,
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/752/files